### PR TITLE
fix: Disable autostart

### PR DESCRIPTION
**Pull request by kylecarbs**

### CODE REVIEW

**Comment by cmoog** [comment]
we need to remove the education messages as well

File: internal/cmd/envs.go [hunk 1]
```diff
@@ -173,6 +173,7 @@ func createEnvCmd() *cobra.Command {
 coder envs create my-new-env --image ubuntu
 coder envs create my-new-powerful-env --cpu 12 --disk 100 --memory 16 --image ubuntu`,
 		PreRun: func(cmd *cobra.Command, args []string) {
+			// TODO: Enable this once we enable autostart.
 			autoStartInfo()
```
Comment (cmoog) [inline internal/cmd/envs.go:177]:
```suggestion
			// TODO: Enable this once we enable autostart.
			// autoStartInfo()
```

File: internal/cmd/envs.go [hunk 2]
```diff
@@ -633,6 +634,5 @@ func autoStartInfo() {
 	} else {
 		preferencesURI = fmt.Sprintf("%s%s", accessURI, "/preferences?tab=autostart")
 	}
-
 	clog.LogInfo("⚡NEW: Automate daily environment startup", "Visit "+preferencesURI+" to configure your preferred time")
```
Comment (cmoog) [inline internal/cmd/envs.go:637]:
```suggestion
	// clog.LogInfo("⚡NEW: Automate daily environment startup", "Visit "+preferencesURI+" to configure your preferred time")
```

Comment (kylecarbs) [reply internal/cmd/envs.go:637]: This function doesn't get called anymore.

**Comment by vapurrmaid** [comment]
Could an alternative approach here just be a revert of dd7b3a982bd4cf05e8f63ab85e999fbe0261e280 ?

**Review by cmoog — APPROVED** [review]

**Comment by kylecarbs** [comment]
We could. Up to you @vapurrmaid

**Comment by vapurrmaid** [comment]
I'd be down to just revert then revert-revert once appropriate. It's ezpz

**Pull request closed without merging**